debian/ubuntu-advantage-tools.postinst

```diff
@@ -424,6 +424,12 @@ case "$1" in
       fi
       chown root:root /var/log/ubuntu-advantage.log
 
+      if dpkg --compare-versions "$PREVIOUS_PKG_VER" lt "31~"; then
+        if [ -f /var/lib/ubuntu-advantage/jobs-status.json ]; then
+            chmod 0644 /var/lib/ubuntu-advantage/jobs-status.json
+        fi
+      fi
+
```

Comment:
Does this change deserve a postinst transition? I think the next time the file is written it will get the new permissions anyway.

Reply:
Yeah, I think you are right. My concern was that if the user disabled the timer jobs, we would not update the file, but this is not the case. I will remove this bit together with the manual test